pipeline-runner/src/data-ingest/4_Prepare_experiment.r

```diff
@@ -107,7 +107,7 @@ add_custom_config_per_sample <- function(step_fn, config, scdata){
     result_config$auto <- NULL
     result_config$enabled <- NULL
     # Update config with the unisample thresholds
-    config[[paste("sample-", sample, sep = "")]] <- result_config
+    config[[sample]] <- result_config
```

Comment:
Is it correct to remove the sample- from the config? Might that break something?

Reply:
It doesn't seem to. There may have been undiagnosed bugs without this change as the pipeline [grabs per-sample config](https://github.com/biomage-ltd/pipeline/blob/9af52b111ce1589994bf4c0b1d0304c94f6b66f5/pipeline-runner/src/init.r#L168) using the `sample_id` directly without the `sample-` prefix. Since it is not created currently, it is initually `NULL`. Only because `auto` is initially `TRUE`, the non `sample-` prefixed config is created during each step of the QC pipeline and then created after being [sent to the API](https://github.com/biomage-ltd/pipeline/blob/9af52b111ce1589994bf4c0b1d0304c94f6b66f5/pipeline-runner/src/handle_data.r#L20).

I tested locally by uploading Vicky's dataset without issue.

Reply:
@aerlaut - @JuanluOnieva-biomage mentioned that you sketched out the requirements when the `sample-` prefix was set up. There seems to have been a regression when data-ingest was moved over to the pipeline and the `sample-` prefix is not currently used (and hasn't been for a while now). Is there any critical reason for having a `sample-` prefix or if we just continue to use only the sample_id hash is that okay?

Reply:
Hey! I've actually realized today separately that this is a necessary change because it led to a weird accumulation of per samples config in the processingConfig and I raised a PR with this change, having forgotten that I saw the same thing in your PR. It'd still be good to have Agi's input on this and the plotdata but I think we should be good to go with this change.

Reply:
That is the rationale for the change. Juanlu pointed out though that we can also change the upload_to_aws.R step so that the initial sample's are named with `sample-` prefixes. This would also fix the accumulation of configs but be in line with @aerlaut original sketchup.